.changelog/25453.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+drivers: set -1 exit code in case of executor failure
```

Comment:
Because this is a function of the driver and not the executor, it won't impact any external drivers that use the executor (our lack of an "executor SDK" aside for the moment). Let's call out the specific drivers we're fixing:

```suggestion
drivers: set -1 exit code in case of executor failure for the exec, raw_exec, java, and qemu task drivers
```